common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -361,7 +361,7 @@ def _create(cls, target_class, **kwargs):  # lint-amnesty, pylint: disable=argum
         location = kwargs.pop('location')
         user_id = kwargs.pop('user_id', ModuleStoreEnum.UserID.test)
         publish_item = kwargs.pop('publish_item', True)
-        has_score = kwargs.pop('has_score', None)
+        has_score = kwargs.get('has_score')
```

Comment:
This fixes an issue I noticed with ItemFactory that might have gotten broken in February where has_score wasn't always getting saved properly for (only lti block?) test items because the kwarg wasn't being passed down to the modulestore creation step.